project/make.sh

```diff
@@ -58,6 +58,8 @@ DEFAULT_BUNDLES=(
 	cover
 	cross
 	tgz
+
+	fedora
```

Comment:
Oh yeah for sure -- sorry if it sounded like I was trying to say that should be smushed into this one too. :no_mouth: